### PR TITLE
Fix directory package mismatch

### PR DESCRIPTION
**Pull request by joejstuart**
Fix linter issue directory-package-mismatch. This is only the Tekton package so far.

https://issues.redhat.com/browse/EC-867

### CODE REVIEW

**Comment by zregvart** [comment]
Commented in JIRA, and saw the PR here, there is a auto-fix in [new version of Regal](https://github.com/StyraInc/regal/releases/tag/v0.28.0), perhaps it would help

File: policy/lib/tekton/recorded_att_data_test.rego [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ** Do not edit this file. Regenerate it using `make sync-test-data` **
 
-package lib.tkn.recorded_att_test
+package lib.tekton
```
Comment (zregvart) [inline policy/lib/tekton/recorded_att_data_test.rego:3]: `make sync-test-data` reverts this, needs a corresponding change here: https://github.com/enterprise-contract/ec-policies/blob/cd89ed963a1e664520e7420063adbc3be9c669dd/Makefile#L156

**Review by zregvart — APPROVED** [review]
sed worked overtime, LGTM!

File: policy/lib/tekton/recorded_att_data_test.rego (hunk 1 above)
Comment (joejstuart) [reply policy/lib/tekton/recorded_att_data_test.rego:3]: I knew that and still forgot to make the change. Thanks.

**Pull request merged**